azure-pipelines.yml

```diff
@@ -47,10 +47,25 @@ jobs:
         testRunTitle: 'Publish test results for Python $(python.version) $(manylinux_flag)'
 
     - task: PublishCodeCoverageResults@1
-      inputs: 
+      inputs:
         codeCoverageTool: Cobertura
         summaryFileLocation: '$(System.DefaultWorkingDirectory)/*coverage.xml'
 
+    - script: |
+        python -m pip install twine
+        make dist
+      displayName: 'Create packages'
+
+    - task: TwineAuthenticate@0
+      inputs:
+        artifactFeeds: TODOproj/TODOfeed
```

Comment:
Think this depends on https://docs.microsoft.com/en-us/azure/devops/artifacts/quickstarts/python-packages?view=azure-devops